test/verify/check-docker

```diff
@@ -39,6 +39,14 @@ class TestDocker(MachineCase):
         with b.wait_timeout(5):
             b.wait_popdown("confirmation-dialog")
 
+    def deleteImageConfirm(self):
+        b = self.browser
+        b.wait_popup('delete-image-confirmation-dialog')
+        b.click('#delete-image-confirmation-dialog-confirm')
+        # popdown should be really quick
+        with b.wait_timeout(5):
```

Comment:
My suggestion is to not bother with lowering timeouts in tests. At times our CI is really loaded.

Reply:
Oh it's the same timeout as the `confirm()`, didn't change it

Reply:
Ack, keep it then.